### PR TITLE
Menu fix

### PR DESCRIPTION
**Pull request by akarol**

### CODE REVIEW

File: cfme/web_ui/menu.py [hunk 1]
```diff
@@ -275,12 +275,15 @@ def sections(self):
...
+                        ('clouds_volumes', 'Volumes'),
                         ('clouds_flavors', 'Flavors'),
                         ('clouds_security_groups', 'Security Groups'),
                         ('clouds_instances', 'Instances',
                             self._tree_func_with_grid(
                                 "Instances by Provider", "Instances by Provider")),
-                        ('clouds_stacks', 'Stacks')
+                        ('clouds_stacks', 'Stacks'),
+                        ('clouds_key_pairs', 'Key Pairs'),
+                        ('clouds_object_stores', 'Object Stores')
```
Comment (RonnyPfannschmidt) [inline cfme/web_ui/menu.py]: a little nitpick, please always attach a finalizing comma, so later additions get a cleaner diff display

**Comment by dajoRH** [comment]
Lint report for commit e2c80ca95edfaf4cf98d46bea2c9d34b0527fabf:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 99f7c2d70d68d9213ee909bf078be5e4b4882323:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/web_ui/menu.py (hunk 1 above)
Comment (akarol) [inline cfme/web_ui/menu.py]: @RonnyPfannschmidt  Done!

**Comment by jkrocil** [comment]
LGTM :+1:

**Pull request merged**